app/models/ability.rb

```diff
@@ -42,4 +42,14 @@ def superadmin_permissions
   def superadmin?
     current_user.has_role? :superadmin
   end
+
+  # Override from blacklight-access_controls-0.6.2 to define registered to include having a role on this tenant
+  def user_groups
+    return @user_groups if @user_groups
+
+    @user_groups = default_user_groups
+    @user_groups |= current_user.groups if current_user.respond_to? :groups
+    @user_groups |= ['registered'] if (!current_user.new_record? && current_user.roles.count > 0)
```

Comment:
lots of tracking down, one tiny little change.  The original method is found [here](https://github.com/projectblacklight/blacklight-access_controls/blob/master/lib/blacklight/access_controls/ability.rb#L104) `        @user_groups |= ['registered'] unless current_user.new_record?
`